mypyc/primitives/generic_ops.py

```diff
@@ -200,32 +200,31 @@
 
 # Call callable object with N positional arguments: func(arg1, ..., argN)
 # Arguments are (func, arg1, ..., argN).
-py_call_op = custom_op(
-    arg_types=[object_rprimitive],
-    result_type=object_rprimitive,
-    is_var_arg=True,
+py_call_op = c_custom_op(
+    arg_types=[],
+    return_type=object_rprimitive,
+    c_function_name='PyObject_CallFunctionObjArgs',
     error_kind=ERR_MAGIC,
-    format_str='{dest} = py_call({comma_args})',
-    emit=simple_emit('{dest} = PyObject_CallFunctionObjArgs({comma_args}, NULL);'))
+    var_arg_type=object_rprimitive,
+    extra_int_constant=(0, c_int_rprimitive))
 
 # Call callable object with positional + keyword args: func(*args, **kwargs)
 # Arguments are (func, *args tuple, **kwargs dict).
-py_call_with_kwargs_op = custom_op(
+py_call_with_kwargs_op = c_custom_op(
     arg_types=[object_rprimitive, object_rprimitive, object_rprimitive],
-    result_type=object_rprimitive,
-    error_kind=ERR_MAGIC,
-    format_str='{dest} = py_call_with_kwargs({args[0]}, {args[1]}, {args[2]})',
-    emit=call_emit('PyObject_Call'))
+    return_type=object_rprimitive,
+    c_function_name='PyObject_Call',
+    error_kind=ERR_MAGIC)
 
 # Call method with positional arguments: obj.method(arg1, ...)
 # Arguments are (object, attribute name, arg1, ...).
-py_method_call_op = custom_op(
-    arg_types=[object_rprimitive],
-    result_type=object_rprimitive,
-    is_var_arg=True,
+py_method_call_op = c_custom_op(
+    arg_types=[],
+    return_type=object_rprimitive,
+    c_function_name='CPyObject_CallMethodObjArgs',
     error_kind=ERR_MAGIC,
-    format_str='{dest} = py_method_call({comma_args})',
-    emit=simple_emit('{dest} = CPyObject_CallMethodObjArgs({comma_args}, NULL);'))
+    var_arg_type=object_rprimitive,
+    extra_int_constant=(0, c_int_rprimitive))
```

Comment:
It would be better for `NULL` to be represented as the pointer type, or at least an unsigned integer, not `c_int`.